src/main/flight/gps_rescue.c

```diff
@@ -669,8 +675,10 @@ void descend(void)
         rescueState.intent.targetVelocityCmS = gpsRescueConfig()->rescueGroundspeed * rescueState.intent.proximityToLandingArea;
         // reduce target velocity as we get closer to home. Zero within 2m of home, reducing risk of overshooting.
         // if quad drifts further than 2m away from home, should by then have rotated towards home, so pitch is allowed
-        rescueState.intent.rollAngleLimitDeg = 0.5f * gpsRescueConfig()->maxRescueAngle * rescueState.intent.proximityToLandingArea;
+        rescueState.intent.rollAngleLimitDeg = gpsRescueConfig()->maxRescueAngle * rescueState.intent.proximityToLandingArea;
         // reduce roll capability when closer to home, none within final 2m
+        rescueState.intent.rollAngleLimitDeg *= 0.5f;
```

Comment:
Try to avoid this sequential computation, if feasible. Either use local variable or single expression.

`rollAngleLimitDeg` is inconsistent for short time interval (unless compiler optimizes first write). It is not problem with cooperative multitasking used in BF, but IMO it is still poor design pattern.

Reply:
Done, thanks.